frontend/src/components/allocation/list.js

```diff
@@ -1,53 +1,76 @@
 import React, { PropTypes } from 'react';
+import { DropdownButton } from 'react-bootstrap';
 import NomadLink from '../link';
 import FormatTime from '../format/time';
+import shortUUID from '../../helpers/uuid';
 import { renderDesiredStatus, renderClientStatus } from '../../helpers/render/allocation';
 
 const allocationStatusColors = {
-    complete: 'success',
-    running: 'info',
+    complete: '',
+    running: '',
     lost: 'warning',
     failed: 'danger',
 };
 
+const getAllocationNumberFromName = (allocationName) => {
+    const match = /[\d+]/.exec(allocationName);
+    return match[0];
+};
+
 const AllocationList = ({ allocations, nodes }) =>
   <table className="table table-hover table-striped">
     <thead>
       <tr>
+        <th></th>
         <th>ID</th>
         <th>Job</th>
         <th>Task Group</th>
-        <th>Task</th>
+        <th>Client</th>
         <th>Client Status</th>
-        <th>Desired Status</th>
-        <th>Node</th>
-        <th>Evaluation</th>
-        <th>Time</th>
+        <th>When</th>
+        <th></th>
       </tr>
     </thead>
     <tbody>
       {allocations.map((allocation) => {
           const color = allocationStatusColors[allocation.ClientStatus];
           return (
             <tr className={ color } key={ allocation.ID }>
+              <td>{ renderClientStatus(allocation) }</td>
```

Comment:
Well, that's the problem isn't it, I interpret it differently from someone else so I'm not convinced it conveys the status equally well and the fact that it's at the beginning of the row with no column header to indicate what it means is not intuitive. That's what I meant by 'lost' :-) 

You've said yourself, needing to hover over the icon to find the state is bad UX. We are going to limit the size of the lists anyway. If client status and desired status are different how is an icon going to help over just displaying those values? I understand that UI is really subjective and personal so we need to figure out a way that keeps us both happy :)